Libraries/DirectMLX.h

```diff
@@ -4356,7 +4431,7 @@ namespace dml
             }
 
             // Sanity
-            assert(desc.nodes.size() == m_operatorNodes.size());
+            assert(desc.operatorNodes.size() == m_operatorNodes.size());
             assert(desc.outputEdges.size() == desc.outputCount);
```

Comment:
Should we also add a sanity check for `desc.constantNodes`?

Reply:
That's a good idea.